api/src/main/java/cz/fi/muni/pa165/dto/UserDTO.java

```diff
@@ -0,0 +1,93 @@
+package cz.fi.muni.pa165.dto;
+
+
+import com.fasterxml.jackson.annotation.JsonView;
+import cz.fi.muni.pa165.enums.Role;
+import cz.fi.muni.pa165.views.View;
+
+/**
+ * Created by Erik Macej 433744 , on 19.11.16.
+ *
+ * @author Erik Macej 433744
+ */
+public abstract class UserDTO {
+
+    @JsonView(View.Summary.class)
+    private Long id;
+
+    @JsonView(View.Summary.class)
+    private String firstName;
+
+    @JsonView(View.Summary.class)
+    private String surname;
+
+    @JsonView(View.Summary.class)
+    private String nick;
+
+    @JsonView(View.Summary.class)
+    private String email;
+
+    private String passwordHash;
```

Comment:
Hashe sa v responsoch posielat nebudu, na to je ta JsonView ale pre pracu s UserDto to potrebujes